packages/atomizer/tests/atomizer.js

```diff
@@ -1302,4 +1302,33 @@ describe('Atomizer()', function () {
             ]);
         });
     });
+    describe('Blend modes', function () {
```

Comment:
I think if we are going to start adding tests for rules, we should create a new file called `rules.js` and start adding them there. Then it can be a 1 to 1 relationship with the existing `rules.js` file.

Reply:
yeah, I guess we need to decide if its beneficial as @src-code had mentioned we aren't really testing anything new here, just the values added...

Reply:
We could probably automate the whole thing but looping through the entire rules.js file and running the test for each rule and creating a snapshot or something. I think all of this could be in a separate PR since we can do it for all rules at one time.